theano/sandbox/cuda/basic_ops.py

```diff
@@ -186,6 +186,8 @@ class GpuElemwise(GpuOp):
 
     """
 
+    __props__ = ("scalar_op", )
```

Comment:
If \__props__ is used, it generate some default implementation of \__eq__ \__hash__ and \__str__. Sometimes we don't want to use the default method. So I just checked which one we want to remove and told you so.